src/snovault/elasticsearch/__init__.py

```diff
@@ -16,12 +18,79 @@
 )
 import json
 import sys
+
+from redis import StrictRedis
+
+
 PY2 = sys.version_info.major == 2
 
 
 ROOT_LOG = logging.getLogger('root')
 
 
+class LocalStoreClient():
+    client = None
+    
+    def __init__(self, redis_db=0, local_store=None):
+        if local_store:
+            self.client = StrictRedis(
+                charset="utf-8",
+                decode_responses=True,
+                db=redis_db,
+                host='localhost',
+                port=6379,
```

Comment:
I know we don't have the same registry, is there any other config file we can read this from? if not, we should make this a constant